notebooks/301-tensorflow-training-openvino/301-tensorflow-training-openvino.ipynb

```diff
@@ -98,6 +98,23 @@
     "from tensorflow.keras import layers\n",
     "from tensorflow.keras.models import Sequential"
    ]
+  },
+    {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Enable Intel® oneAPI Deep Neural Network Library (oneDNN) Optimizations\n",
```

Comment:
Do you think this is important enough to make this a header (instead of a comment), considering it will be the default in TF 2.6? I am fine with it if you think it is important enough to call out.

Reply:
I just tested this on Windows and as Raymond mentioned, we need to add the os.environ call before the tensorflow import to get the speedup.

Reply:
Yes, please make it a comment instead of markdown cell. And yes it should be in front of import tensorflow.

Reply:
I'll merge this now and do that in the branch I'll make for the 2021.4 update